bundles/org.openhab.binding.thekeys/src/main/resources/OH-INF/i18n/thekeys.properties

```diff
@@ -0,0 +1,48 @@
+# add-on
+
+addon.thekeys.name = TheKeys Binding
+addon.thekeys.description = TheKeys Binding controls a smartlock through the bluetooth gateway
+
+# thing types
+
+thing-type.thekeys.gateway.label = TheKeys Gateway
+thing-type.thekeys.gateway.description = Gateway controlling a smartlock
+thing-type.thekeys.smartlock.label = TheKeys Smartlock
+thing-type.thekeys.smartlock.description = A smartlock controlled by the gateway
+
+# thing types config
+
+thing-type.config.thekeys.gateway.apiTimeout.label = API Timeout
+thing-type.config.thekeys.gateway.apiTimeout.description = The api timeout for each request
+thing-type.config.thekeys.gateway.code.label = Code
+thing-type.config.thekeys.gateway.code.description = Code to access the gateway
+thing-type.config.thekeys.gateway.host.label = Host
+thing-type.config.thekeys.gateway.host.description = Host or IP address of the gateway
+thing-type.config.thekeys.gateway.refreshInterval.label = Refresh Interval
+thing-type.config.thekeys.gateway.refreshInterval.description = The refresh interval in seconds which is used to poll the gateway
+thing-type.config.thekeys.smartlock.lockId.label = Lock ID
+thing-type.config.thekeys.smartlock.lockId.description = Identifier of the smartlock
+
+# channel types
+
+channel-type.thekeys.lastSync.label = Last Sync
+channel-type.thekeys.lastSync.description = Last success synchronization with the smartlock
+channel-type.thekeys.lock_status.label = Lock Status
+channel-type.thekeys.lock_status.description = Status of the smartlock
```

Comment:
Please regenerate the i18n files, they contain old names for channels.
mvn i18n:generate-default-translations